ROADMAP.md

```diff
@@ -2,10 +2,10 @@
 
 ## TSLint rules
 
-✅ = done  
-🌟 = in ESLint core  
-🔌 = in another plugin  
-🛑 = unimplemented  
+✅ = done
+🌟 = in ESLint core
+🔌 = in another plugin
+🛑 = unimplemented
```

Comment:
Looks like these trailing spaces got accidentally removed. Can you add them back? They force the key to break onto separate lines.

Reply:
Re-added the trailing spaces.